manage-data/lifecycle/index-lifecycle-management/rollover.md

```diff
@@ -9,50 +9,96 @@ products:
-::::
+Decide your approach to index rotation based on your use case and requirements.
 
+| Use case               | Recommended approach                                      | Setup benefits and limitations                                                                  |
+| ---------------------- | --------------------------------------------------------- | ----------------------------------------------------------------------------------------------- |
+| Logs, metrics          | [Data streams](rollover.md#rollover-data-stream)          | Configure rollover with lifecycle management, *minimal setup*, control over rollover timing ^1^ |
+| Legacy indexing setup  | [Alias-based rollover](rollover.md#rollover-with-aliases) | Configure rollover with lifecycle management, *advanced setup*, control over rollover timing    |
+| Small, static datasets | No rollover                                               | Simpler management                                                                              |
 
-::::{important}
-Empty indices will not be rolled over, even if they have an associated `max_age` that would otherwise result in a roll over occurring. A policy can override this behavior, and explicitly opt in to rolling over empty indices, by adding a `"min_docs": 0` condition. This can also be disabled on a cluster-wide basis by setting `indices.lifecycle.rollover.only_if_has_documents` to `false`.
-::::
+^1^ Rollover is handled automatically in {{es-serverless}} projects. {applies_to}`serverless: ga`
 
+:::{tip}
+For new projects, use data streams. They're simple to manage with lifecycle policies where you define phases and actions that handle rollover automatically.
+:::
 
-::::{important}
-The rollover action implicitly always rolls over a data stream or alias if one or more shards contain 200000000 or more documents. Normally a shard will reach 50GB long before it reaches 200M documents, but this isn’t the case for space efficient data sets. Search performance will very likely suffer if a shard contains more than 200M documents. This is the reason of the builtin limit.
-::::
 
+### Rotating your indices with data streams [rollover-data-stream]
+
+We recommend using [data streams](../../data-store/data-streams.md) to manage time series data. When set up to use an ILM policy that includes rollover, a data stream automatically manages the rotation of your indices. This ensures you can write to the data stream without additional configuration.
+When targeting a data stream, each time the current write index reaches a specified age or size, a new backing index is generated (with an incremented number and timestamp), and it becomes the data stream's writing index.
+
+Each data stream requires an [index template](../../data-store/templates.md) that contains the following:
+
+* A name or wildcard (`*`) pattern for the data stream.
+* Optional: The mappings and settings applied to each backing index when it’s created.
```

Comment:
I think we're missing the `"data_stream": {}` part of the template:

```suggestion
* A name or wildcard (`*`) pattern for the data stream.
* A configuration that indicates a data stream should be used for the index pattern instead of a concrete index.
* Optional: The mappings and settings applied to each backing index when it’s created.
```